lib/suspenders/adapters/heroku.rb

```
@@ -34,6 +33,46 @@ def set_heroku_rails_secrets
         end
       end
 
+      def provide_review_apps_setup_script
+        app_builder.template(
+          "bin_setup_review_app.erb",
+          "bin/setup_review_app",
+          force: true,
+        )
+        app_builder.run "chmod a+x bin/setup_review_app"
+      end
+
+      def create_heroku_pipelines_config_file
+        template "app.json.erb", "app.json"
+      end
+
+      def create_heroku_pipeline
+        heroku_app_name = app_name.dasherize
+
+        pipelines_plugin = `heroku plugins | grep pipelines`
+        if pipelines_plugin.empty?
+          puts 'You need heroku pipelines plugin. Run: heroku plugins:install heroku-pipelines'
+          exit 1
```

@tute We should fail if a subcommand fails, but I don't consider the user missing a plugin that doesn't impact the use of suspenders a valid reason for failing the whole generation.